react/src/components/SettingList.tsx

```diff
@@ -87,6 +88,15 @@ const GroupSettingItems: React.FC<{
           {group.title}
         </Typography.Title>
         <Divider style={{ marginTop: 0, marginBottom: 0 }} />
+        {group.description && (
+          <div
+            style={{
+              marginTop: token.marginSM,
+            }}
+          >
+            {group.description}
+          </div>
+        )}
```

Comment:
How about secondary type Text for string type description?
```suggestion
          <Typography.Text
            type="secondary"
            style={{
              marginTop: token.marginSM,
            }}
          >
            {group.description}
          </Typography.Text>
        )}
```